src/Tests/dotnet-workload-install.Tests/GivenDotnetWorkloadInstall.cs

```diff
@@ -266,6 +267,33 @@ public void GivenWorkloadInstallItDoesNotRemoveOldInstallsOnRollback()
             installPacks.Count().Should().Be(3);
         }
 
+        [Fact]
+        public void GivenWorkloadInstallItFallsBackToPriorFeatureBands()
```

Comment:
Is this giving us any extra coverage that's not already going to be covered by the SdkDirectoryWorkloadManifestProviderTests?  I'm not sure whether we need this test or not.